documentation/content/en/articles/committers-guide/_index.adoc

```diff
@@ -2019,6 +2019,93 @@ git log refs/freebsd/vendor/zlib/1.2.10
 ....
 
 would look at the log for the vendor branch for zlib starting at 1.2.10.
+=== Collaborating with Others
+
+One of the keys to good software development on a project as large as FreeBSD is the ability to collaborate with others before you push your changes to the tree.  The FreeBSD project's git repositories do not, yet, allow user created branches to be pushed to the repository, and therefore if you wish to share your changes with others you must use another mechanism, such as a hosted GitLab or github, in order to share changes in a user generate branch.
```

Comment:
*generated

please also spell it GitHub here and everywhere

(I'm not a doc person): not sure about line length and wrapping also